packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.tsx

```diff
@@ -459,6 +459,7 @@ export class ContextualMenu extends BaseComponent<IContextualMenuProps, IContext
 
     if (ev.which === openKey) {
       this._onItemSubMenuExpand(item, ev.currentTarget as HTMLElement);
+      ev.preventDefault();
```

Comment:
I think in my mind, if you don't stop propagation, it means that a component higher up the dom tree can do things which aren't expected.

E.g. you nest a dropdown inside of a dialog, both handle the escape button to dismiss. If you don't stop propagation, then pressing escape while the dropdown is expanded also dismisses the dialog.

You could look at `ev.defaultPrevented`, i just haven't really seen that in practice on keyboard handlers since it's not intuitive to predict you may need to do that. Maybe that is a pattern we need to introduce?